deployment/eclipse-kuksa/utils/hono-influxdb-connector/kubernetes/deployment.yaml

```diff
@@ -26,6 +26,7 @@ spec:
     spec:
       containers:
       - image: <DOCKER_IMAGE>:<DOCKER_VERSION>
+        imagePullPolicy: Always
```

Comment:
Is there a specific reason why "Always" is necessary? To me "IfNotPresent" could be sufficient. (https://kubernetes.io/docs/concepts/configuration/overview/#container-images)

Reply:
Always helps when we re-build an image without changing the tag. Of course only the check will happen "always". If the image has not been updated, it will not be pulled again.